frontend/src/components/pages/EditAccountPage.tsx

```diff
@@ -207,26 +217,45 @@ const EditAccountPage = (): React.ReactElement => {
           Edit Profile
         </Text>
         <ProfilePhotoForm />
-        <Divider my={8} />
+        {user && (
+          <HStack mt={6}>
+            {user.branches.map((branch) => (
+              <Tag
+                size="lg"
+                key="lg"
+                borderRadius="full"
+                variant="brand"
+                colorScheme="red"
+              >
+                <TagLabel>{branch.name}</TagLabel>
+              </Tag>
+            ))}
+          </HStack>
+        )}
+        <Divider my={6} />
         {user && (
           <AccountForm
             key={key}
             mode={AccountFormMode.EDIT}
             isAdmin={authenticatedUser?.role !== Role.Volunteer}
-            firstName={user?.firstName}
-            lastName={user?.lastName}
+            firstName={user.firstName}
+            lastName={user.lastName}
             email={user?.email}
-            dateOfBirth={moment(user?.dateOfBirth).format("YYYY-MM-DD")}
-            pronouns={user?.pronouns}
-            phoneNumber={user?.phoneNumber}
-            emergencyNumber={user?.emergencyContactPhone}
-            emergencyEmail={user?.emergencyContactEmail}
-            emergencyName={user?.emergencyContactName}
-            prevLanguages={user?.languages?.map((language) => ({
+            dateOfBirth={
+              user.dateOfBirth
+                ? moment(user?.dateOfBirth).format("YYYY-MM-DD")
```

Comment:
Nit: optional chaining not needed anymore

```suggestion
                ? moment(user.dateOfBirth).format("YYYY-MM-DD")
```

Reply:
I added this ternary statement because `user.dateOfBirth` can be null, and so moment complains in the dev console when it receives null as a parameter. Should I still change this?